launchers/launcher.go

```diff
@@ -114,6 +114,11 @@ func LaunchModules(ctx context.Context, modules config.ServerType) (context.Canc
 		if err != nil {
 			return shutdownCancel, err
 		}
+
+		log.Debug("Finishing origin server configuration")
```

Comment:
This needs to move back to the original location: the web engine needs to have started up for `OriginServeFinish` to run (as this will potentially perform API requests against the director APIs).

Reply:
Please remove this dangling log line.